public/styles.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: helvetica, arial
+}
+h1 {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+input {
+  height: 2em;
+  border: 1px solid;
+  border-color: #AAAAAA;
```

Comment:
You could write the border proprieties all together. And when the color it's all the same you can use just 3 values, so would be better something like this `border: 1px solid #aaa;`